src/install/browserFetcher.ts

```diff
@@ -109,10 +108,10 @@ export async function downloadBrowserWithProgressBar(browserPath: string, browse
   }
 
   const url = revisionURL(browser);
-  const zipPath = path.join(os.tmpdir(), `playwright-download-${browser.name}-${browserPaths.hostPlatform}-${browser.revision}.zip`);
+  const zipPath = browserPaths.browserZipFile(browserPath, browser);
```

Comment:
we explicitly do not support concurrent access to this folder - it would require lockdirs with pid files and all the fun. We'd rather stay away from this world for now.